sdk/identity/azure-identity/test/live/environment_credential.cpp

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#include <azure/identity/environment_credential.hpp>
+
+#include <gtest/gtest.h>
+
+#include <chrono>
+
+TEST(EnvironmentCredential, ClientSecret)
+{
+  Azure::Identity::EnvironmentCredential const credential;
+
+  auto const token = credential.GetToken(
```

Comment:
Can you add some comments if the EnvironmentCredential would expect to find some specific env variables in the system?

Reply:
There is already comment for that: https://github.com/Azure/azure-sdk-for-cpp/blob/master/sdk/identity/azure-identity/inc/azure/identity/environment_credential.hpp#L27-L33

Reply:
yes, but that's inside the Class. 
What I would like to have is a quick way to know what do I need to run and make this test pass.
I is fine also, if the test would fail and display a text like "Missing env variables" and then list the expected or required env vars.

Otherwise, if I am some contributor trying to run this test and it fails, it might take me a while to get to the class where the hint about the requirements are.

You can also just add a simple hit: "Look a this link for the test requirements: ...." and use the link you shared

Reply:
I'll better add an `EXPECT_FALSE(GetEnv("AZURE_...").empty())`, before constructing the credential.